html/captive-portal/lib/captiveportal/PacketFence/Controller/Activate/Email.pm

```diff
@@ -226,6 +228,7 @@ sub doSponsorRegistration : Private {
 
         # send to a success page
         $c->stash(
+            title => "Sponsor request accepted",
```

Comment:
i18n or directly in the template IMO

The configuration will get heavy for something that is never changed